### PR TITLE
Modify tools section for earth-system

### PR DESCRIPTION
**Pull request by Marie59**
Hi everyone ! I tried (following single-cells example) to add earth-system own sections for the subdomain and to modify the interactive tools section for my subdomain. I am pretty sure there should be a better/cleaner way to do it. Tell me what do you think of it. This is not urgent and if what I did is not good it can wait :)
Thank you !!

### CODE REVIEW

File: templates/galaxy/config/global_host_filters.py.j2
Comment (kysrpex) [inline templates/galaxy/config/global_host_filters.py.j2:145]:
```suggestion
    ],
```

File: templates/galaxy/config/tool_conf.xml.j2
Comment (kysrpex) [inline templates/galaxy/config/tool_conf.xml.j2:607]:
```suggestion
  <!-- Earth-system subdomain -->
<label id="earth-system-label-interactive-tools" text="Interactive tools" />  
```

**Review by kysrpex — COMMENTED** [review]
Actually, especially if you only need interactive tools, this is not a bad idea. The story would be much different if you also wanted to control which non-interactive tools are shown.

Please read my comments and let me know when you consider you are done.

**Comment by Marie59** [comment]
Thank you for the review ! I think I am done @kysrpex .

**Review by kysrpex — APPROVED** [review]
Great :)

If you don't mind I will wait until tomorrow morning to merge this. I am pretty sure it will just work, but it is not wise to risk breaking something just before you leave the office, specially if it is not urgent.

**Comment by Marie59** [comment]
> Great :)
> 
> If you don't mind I will wait until tomorrow morning to merge this. I am pretty sure it will just work, but it is not wise to risk breaking something just before you leave the office, specially if it is not urgent.

Ahah yes of course you are totally right ! Have a nice evening :)

**Pull request merged**
